frame/example-offchain-worker/src/lib.rs

```diff
@@ -153,7 +157,7 @@ impl<T: Trait> Module<T> {
 	fn send_to_sc_mock() -> Result<(), &'static str> {
 
 		debug::info!("[OCW] Getting signer");
-		let signer = Signer::<T::CST, T::AuthorityId>::all_accounts();
+		let signer = Signer::<T::CST, T::AuthorityId>::any_account();
```

Comment:
any_account to send only one transaction, even if there are multiple accounts on the node.